src/views/SecondAssessment.vue

```diff
@@ -14,16 +13,16 @@ const quizStore = useQuizStore()
 
 onMounted(async () => {
   role.value = await rolesStore.getRoleById(roleId.value)
-  quizStore.initializeQuizAnswers(role.value.knowledges ?? [], quizAnswer.value)
+  quizStore.initializeQuizAnswers(role.value.knowledges ?? [], quizStore.quizAnswers.value)
```

Comment:
You don't need to pass this second parameter, it's accessible inside the function